frontend/src/components/Shared/Navbar.jsx

```
@@ -1,154 +1,189 @@
+                            >
+                                Sign Up
+                            </NavLink>
+                        </>
+                    ) : (
+                        <button
+                            onClick={() => {
+                                logoutUser();
+                                setIsMenuOpen(false);
+                            }}
+                            className="btn bg-blue-500 hover:bg-blue-600 border-0 text-white cursor-pointer"
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)